### PR TITLE
Loookasz vuerouter

### PR DESCRIPTION
**Pull request by looookasz**
made links between pages

### CODE REVIEW

File: client/src/App.vue [hunk 1]
```diff
@@ -13,5 +13,15 @@ export default {
...
+<style>
+.RouterLink {
+  text-decoration: none;
+  color: #000000 !important;
+}
+.toolbar-title {
+  text-decoration: none;
+  color: #ffffff !important;
+}
+.router-link-active {
```
Comment (wojtasiq) [inline client/src/App.vue:25]: empty style?

File: client/src/App.vue [hunk 2]
```diff
@@ -13,5 +13,15 @@ export default {
   }
 };
 </script>
-
-<style></style>
+<style>
```
Comment (wojtasiq) [inline client/src/App.vue:16]: please add scoped keyword ;)

Comment (Isild) [reply client/src/App.vue:16]: @looookasz

**Comment by Isild** [comment]
@looookasz add scoped keyword as Wojtek said and it will be good

File: client/src/App.vue (hunk 1 above)
Comment (looookasz) [reply client/src/App.vue:25]: ^removing underline and color from links

**Pull request merged**